packages/subgraph/schema.graphql

```diff
@@ -598,9 +639,15 @@ type RewardAddressChangedEvent implements Event @entity {
 type PPPConfigurationChangedEvent implements Event @entity {
     id: ID!
     transactionHash: Bytes!
+    gasPrice: BigInt!
     timestamp: BigInt!
     name: String!
 
+    """
+    The address of the governance contract the event was emitted from.
+    """
+    governanceAddress: Bytes!
```

Comment:
I didn't see this mapped in: `packages/sdk-core/src/mapGetAllEventsQueryEvents.ts`